tools/fds/server.cpp

```diff
@@ -99,15 +104,79 @@ int fastdds_discovery_server(
 
     DomainParticipantQos participantQos;
 
+    if (nullptr != options[XML_FILE])
+    {
+        sXMLConfigFile = options[XML_FILE].arg;
+        if (sXMLConfigFile.length() > 0)
+        {
+            size_t delimiter_pos = sXMLConfigFile.find(delimiter);
+            if (std::string::npos != delimiter_pos)
+            {
+                profile = sXMLConfigFile.substr(delimiter_pos + 1, sXMLConfigFile.length());
```

Comment:
If we change the profile position, this should be changed

Reply:
Done in 6dd1ad92f